Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -4958,7 +4958,7 @@ final class BuildPlanTests: XCTestCase {
 
         let yamlContents: String = try fs.readFileContents(yaml)
         let inputs = """
-            inputs: ["\(AbsolutePath("/Pkg/Snippets/ASnippet.swift"))","\(swiftGetVersionFilePath.escapedPathString)","\(AbsolutePath("/Pkg/.build/debug/Lib.swiftmodule"))"
+            inputs: ["\(AbsolutePath("/Pkg/Snippets/ASnippet.swift").escapedPathString)","\(swiftGetVersionFilePath.escapedPathString)","\(AbsolutePath("/Pkg/.build/debug/Lib.swiftmodule").escapedPathString)"
         """
         XCTAssertMatch(yamlContents, .contains(inputs))
```

Comment:
We may need to make `underlying` public on `SerializedJSON for this to work.

```suggestion
        XCTAssertMatch(yamlContents, .contains(inputs.underlying))
```